index.android.js

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {AppRegistry} from 'react-native';
+
+/* eslint-disable no-unused-vars */
+import Mattermost from 'app/mattermost';
```

Comment:
Instead of ignoring the error, you can just do `import 'app/mattermost';`

Reply:
right, that should do it

Reply:
actually we can't get rid of that one as we need to instantiate the app with `new Mattermost()` so is either a no-unused-vars or a no-new rule that needs to be disabled

Reply:
Oh, I didn't notice we used both. I don't see how we have an unused variable here then since they're both clearly used

Reply:
it is for the `const app = new Mattermost()` app is not being used anywhere